swarm/network/kademlia.go

```diff
@@ -735,8 +735,18 @@ func NewPeerPotMap(neighbourhoodSize int, addrs [][]byte) map[string]*PeerPot {
 	return ppmap
 }
 
-// saturation returns the smallest po value in which the node has less than MinBinSize peers
+// Saturation returns the smallest po value in which the node has less than MinBinSize peers
 // if the iterator reaches neighbourhood radius, then the last bin + 1 is returned
+func (k *Kademlia) Saturation() int {
+	k.lock.RLock()
+	defer k.lock.RUnlock()
+
+	return k.saturation()
+}
+
+// saturation returns the smallest po value in which the node has less than MinBinSize peers
```

Comment:
As we duplicate the godoc comment and comments are not required for private functions. I would just drop the private duplicate. => We ~half the chances of inconsistent comments. :)